jcl/src/java.base/share/classes/java/lang/StringBuffer.java

```diff
@@ -529,8 +529,59 @@ public StringBuffer append (float value) {
  * @param		value	the integer
  * @return		this StringBuffer
  */
-public StringBuffer append (int value) {
-	return append(Integer.toString(value));
+public synchronized StringBuffer append(int value) {
+	/*[IF Sidecar19-SE]*/
+	/*[IF Sidecar19-SE-B174]*/
+	int currentLength = lengthInternalUnsynchronized();
+	int currentCapacity = capacityInternal();
+			
+	int valueLength = Integer.stringSize(value);
+			
+	int newLength = currentLength + valueLength;
+			
+	if (newLength > currentCapacity) {
+		ensureCapacityImpl(newLength);
+	}
+
+	if (String.enableCompression && count >= 0) {
+		Integer.getChars(value, newLength, this.value);
+	} else {
+		StringUTF16.getChars(value, newLength, this.value);
+	}
+
+	count = newLength;
+			
+	return this;
+	/*[ELSE]*/
+	return append (String.valueOf (value));
```

Comment:
Note to reviewers that b148 JCL does not have the `StringUTF16.getChars(II[B)` overload and hence we cannot use the above code to carry out the `append` operation.

Reply:
Why the switch from `Integer.toString` to `String.valueOf`?

Reply:
This is a mistake. Fixed in bddd9fa